packages/components/dashboard/details-card/details-card.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { HealthBody } from '@openshift-console/dynamic-plugin-sdk-internal';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Divider,
+  Gallery,
+  GalleryItem,
+  Skeleton,
+} from '@patternfly/react-core';
+import { DATA_FEDERATION_NAMESPACE, MCG_DEPLOYER } from '../../../constants';
+import { ClusterServiceVersionModel } from '../../../models';
+import { ClusterServiceVersionKind } from '../../../types';
+import { referenceForModel } from '../../../utils';
+import ResourceLink from '../../../utils/generics/resource-link';
+import { operatorResource } from '../../resources';
+import './details-card.scss';
+
+export const DetailsCard: React.FC = () => {
+  const { t } = useTranslation();
+  const [csvData, csvLoaded, csvError] =
+    useK8sWatchResource<ClusterServiceVersionKind[]>(operatorResource);
+  const operator = React.useMemo(
+    () => csvData?.find((csv) => csv?.metadata?.name.startsWith(MCG_DEPLOYER)),
+    [csvData]
+  );
+  const operatorVersion = operator?.spec?.version;
+  const operatorName = operator?.metadata?.name;
+  const path = `/k8s/ns/${DATA_FEDERATION_NAMESPACE}/${referenceForModel(
+    ClusterServiceVersionModel
+  )}/${operatorName}`;
+  return (
+    <Card>
+      <CardHeader>
+        <CardTitle className="details-card-title">{t('Details')}</CardTitle>
+      </CardHeader>
+      <Divider />
```

Comment:
```suggestion
```

Reply:
I think as per UX we need to add divider https://marvelapp.com/prototype/2e221bb6/screen/85673714

I missed it in status, adding so in this PR

Reply:
no need for divider...
we don't blindly follow UX, we can add minor changes to keep consistency with other places...

Reply:
ok sure got it